### ISSUE TITLE
Cannot get the USB recognized properly by the teaching pendant

### ISSUE THREAD

**Issue by Rodrigo-Ramon** [UniversalRobots/Universal_Robots_ExternalControl_URCap#30]
Hello, 

I'm trying to follow the guidance here and on the main UR-ROS documentation pages but I'm not sure what I'm doing wrong. 

When I use an almost brand new USB stick, load the URCAP file and plug it into the teaching pendant, it claims that it is improperly formatted, but when I confirm I would like it to properly format it, it takes 2 seconds and claims it is done. Unplugging and plugging the USB yields the same error. When I plug it into a windows machine, the folder is no longer pulled up. Plugging the "formatted" USB into a linux machine shows me the original folder with no formatting having been performed.

Would you have some guidance on how to get this to be recognized by the TP?

Any help is appreciated. Thank you

**Comment by padit1337** [comment]
I don't think this is the right place for your question, non the less, you should figure out if your USB Drive is formatet as FAT32 exFAT or NTFS. Normally I would expect exFAT to yield the best results. If you format it to exFAT and still run into this problem, please a quick update and describe how you formated the drive.

**Comment by urfeex** [comment]
@Rodrigo-Ramon I will go ahead and close this since you seem to have gotten a valid response in the meantime. If your problem still exists, please let us know.
